plugins/source/gcp/docs/tables/gcp_websecurityscanner_findings.md

```diff
@@ -0,0 +1,38 @@
+# Table: gcp_websecurityscanner_findings
+
+https://cloud.google.com/security-command-center/docs/reference/web-security-scanner/rest/v1/projects.scanConfigs.scanRuns.findings
+
+The composite primary key for this table is (**project_id**, **scan_run_name**, **name**).
```

Comment:
Do we need `scan_run_name` as a PK?

Seems like it's included in the name based on https://cloud.google.com/security-command-center/docs/reference/web-security-scanner/rest/v1/projects.scanConfigs.scanRuns.findings#Finding